File: app/src/main/java/com/bianhua/com/fragment/HomeFragment.java
package com.bianhua.com.fragment;

import android.widget.TextView;

import com.bianhua.com.R;
import com.bianhua.com.base.WDFragment;
import com.bianhua.com.bean.SendMessageCommunitor;

import butterknife.BindView;
import butterknife.OnClick;

/**
 * @author 聂俊鹏
 * @date 首页
 * qq:2241659414
 */
public class HomeFragment extends WDFragment {


    @BindView(R.id.text_chuan)
    TextView textChuan;
    private SendMessageCommunitor sendMessage;

    @Override
    protected int getLayoutId() {
        return R.layout.fragment_home;
    }

    @Override
    protected void initView() {

    }

    @OnClick(R.id.text_chuan)
    public void onViewClicked() {

        sendMessage.sendMessage("mainselecet1");
    }
}

File: app/src/main/java/com/bianhua/com/fragment/HomeFragment.kt
package com.bianhua.com.fragment

import android.os.Bundle
import android.view.View
import android.widget.Toast
import com.bianhua.com.R
import com.bianhua.com.Utils.DiaLog_Jihe.ViewUtils
import com.bianhua.com.base.WDFragment
import com.bianhua.com.bean.SendMessageCommunitor
import kotlinx.android.synthetic.main.fragment_home.*
import kotlinx.android.synthetic.main.title_layout.*

/**
 * @author 聂俊鹏
 * @date 首页         第一个fragment
 * qq:2241659414
 */
class HomeFragment : WDFragment(){



    override val layoutId: Int = R.layout.fragment_home

    override fun onResume() {
        super.onResume()
        title_text.setText("首页")


    }

}